build_test/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
-## 1.2.1-dev
+## 1.2.1
```

Comment:
Should this be `1.3.0`? Seems like a new feature to me...

Reply:
It is a bug fix imo - it fixes opted out tests with the experiment enabled.